### DISCUSSION TITLE
How to import XML to show only values and not tags?

### DISCUSSION THREAD

**Discussion by reregaga** [freeplane/freeplane#800] [Help]
I'm trying to import this XML file into Freeplane:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<breakfast_menu>
  <food>
    <name>Belgian Waffles</name>
    <price>$5.95</price>
    <description>Two of our famous Belgian Waffles with plenty of real maple syrup</description>
    <calories>650</calories>
  </food>
  <food>
    <name>Strawberry Belgian Waffles</name>
    <price>$7.95</price>
    <description>Light Belgian waffles covered with strawberries and whipped cream</description>
    <calories>900</calories>
  </food>
</breakfast_menu>
```
I expect to see one root node and a food list. But instead I see a lot of xml tags:
>![xml](https://user-images.githubusercontent.com/76629445/200033902-a1a51f59-dd88-4102-a889-1c6be1a85443.png)

This is useful in some use cases, but I would like to get a simple single level list, just the values of nodes like this:
>![attr](https://user-images.githubusercontent.com/76629445/200034649-a9ad8d3e-7408-478b-b361-e5e7283c5f49.png)

or this:
>![xml](https://user-images.githubusercontent.com/76629445/200034778-ecb0ec7b-1bf4-4d6b-86ac-0d1f3e779a26.png)

 How to do it? Are there any import settings or will help only XSLT or Groovy script?

**Comment by ckillerz** [comment]
solution is to change the xml format

```
<?xml version="1.0" encoding="UTF-8"?>
<breakfast_menu>
  <food name="Belgian Waffles" price="$5.95" description="Two of our famous Belgian Waffles with plenty of real maple syrup" calories="650" />
  <food name="Strawberry Belgian Waffles" price="$7.95" description="Light Belgian waffles covered with strawberries and whipped cream" calories="900" />
</breakfast_menu>
```
